system-variables.md

```diff
@@ -429,6 +429,14 @@ This variable is an alias for `last_insert_id`.
 - Default value: `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
 - This variable controls a number of MySQL compatibility behaviors. See [SQL Mode](/sql-mode.md) for more information.
 
+### sql_require_primary_key <span class="version-mark">New in TBD</span>
```

Comment:
TBD, can/should we set a version yet?

Reply:
I saw with some other sysvars we included versions, I don't know how docs for different versions are rendered on the website, but I wouldn't want this sysvar doc to appear on the website for a version in which its not available. If you think that's ensured by how they are rendered, then I am happy to drop this.

Reply:
The most likely scenario is the docs team will keep this PR open until the version is known. In the last few weeks prior to release they also pause the script which publishes the docs.

But I think you can state it as v6.3, since presumably this is the release to follow v6.2.

Reply:
Sounds good to me. I will put in v6.3